core/src/test/scala/integration/kafka/network/DynamicConnectionQuotaTest.scala

```diff
@@ -202,6 +208,7 @@ class DynamicConnectionQuotaTest extends BaseRequestTest {
     val futures = newListenerNames.map { listener =>
       executor.submit((() => verifyConnectionRate(3, listenerConnRateLimit, listener)): Runnable)
     }
+    waitForConnectionCloseToInitState(initialConnectionCount)
```

Comment:
Shouldn't we put this after waiting for the futures (next line)?

Reply:
Oh, man, thanks for reminding! Fixed. Thanks.